data_platform/preprocessing/preprocess_reddit.py

```
@@ -86,17 +101,78 @@ def passes_row_validators(
+    """
+    if not config_path.is_file():
+        raise FileNotFoundError(config_path)
+    config = load_yaml_config(config_path)
+    dataset_id = require_dataset_id(config, platform="reddit")
+    preprocess = config.get(PREPROCESS_KEY)
+    if not isinstance(preprocess, dict):
+        raise ValueError("dump config must include preprocess")
+    sample_size = preprocess.get(SAMPLE_SIZE_KEY)
+    sample_seed = preprocess.get(SAMPLE_SEED_KEY)
+    if not isinstance(sample_size, int) or sample_size < MIN_SAMPLE_SIZE:
+        raise ValueError("sample_size must be at least 1")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)